flash/image/data.py

```diff
@@ -60,7 +60,7 @@ def image_loader(filepath: str):
 
 class ImageDeserializer(Deserializer):
```

Comment:
Maybe this naming would be better:
```suggestion
class ImageServeInput(ServeInput):
```

Reply:
Let's do this when we get rid of the Deserializer type entirely.